msmtools/generation/api.py

```diff
@@ -131,22 +131,30 @@ def trajectory(self, N, start=None, stop=None):
 
     def trajectories(self, M, N, start=None, stop=None):
         """
-        Generates M trajectories, each of length N, starting from state s
+        Generates M trajectories, each of length N, starting from state(s) s
 
         Parameters
         ----------
         M : int
             number of trajectories
         N : int
             trajectory length
-        start : int, optional, default = None
-            starting state. If not given, will sample from the stationary distribution of P
+        start : int or ndarray, optional, default = None
+            starting state(s). If not given, will sample from the stationary distribution of P.
+            If type(start) == int, the same starting state will be used to generate the M trajectories,
+            if type(start) == ndarray, the i-th starting state from that array is taken trajectory generation.
         stop : int or int-array-like, optional, default = None
             stopping set. If given, the trajectory will be stopped before N steps
             once a state of the stop set is reached
 
         """
-        trajs = [self.trajectory(N, start=start, stop=stop) for _ in range(M)]
+
+        #ToDo If start is an ndarray, ensure that it is 1 dimensional and len(start) >= M
```

Comment:
please implement this check, raise a ValueError, if the given start array is too small.